scripts/aws-datastore/src/androidTest/res/raw/amplifyconfigurationupdated.json

```diff
@@ -0,0 +1 @@
+/Users/dengdan/.aws-amplify/amplify-android/aws-datastore/src/androidTest/res/raw/amplifyconfigurationupdated.json
```

Comment:
please delete -- we might want to remove this file (symlink?) from the cloud so that this doesn't accidentally get committed in the future

Reply:
and/or adding to .gitignore